subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,114 @@
-+
-Note that Gradle isn't limited to building just JVM projects, and it even comes packaged with support for building native projects.
+Gradle runs on the JVM. This is a bonus for users familiar with Java, since build logic can use the standard Java APIs.
+It also makes it easy to run Gradle on different platforms.
 Conventions::
-Gradle takes a leaf out of Maven's book and makes common types of projects — such as Java projects — easy to build by implementing conventions. Apply the appropriate plugins and you can easily end up with slim build scripts for many projects. But these conventions don't limit you: Gradle allows you to override them, add your own tasks, and make many other customizations to your convention-based builds.
+Gradle makes common types of projects easy to build through conventions. Plugins set sensible defaults to keep build scripts minimal.
+But these conventions don't limit you: you can configure settings, add your own tasks, and make many other customizations in your builds.
 Extensibility::
-You can readily extend Gradle to provide your own task types or even build model. See the Android build support for an example of this: it adds many new build concepts such as flavors and build types.
+You can readily extend Gradle to provide your own build logic with custom tasks and plugins.
+See Android builds for an example: they add many new build concepts such as flavors and build types.
 IDE support::
-Several major IDEs allow you to import Gradle builds and interact with them: Android Studio, IntelliJ IDEA, Eclipse, and NetBeans. Gradle also has support for generating the solution files required to load a project into Visual Studio.
+Several major IDEs provide interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, VSCode, and NetBeans.
+Gradle can also generate the solution files required to load a project into Visual Studio.
 Insight::
-https://scans.gradle.com/[Build scans] provide extensive information about a build run that you can use to identify build issues. They are particularly good at helping you to identify problems with a build's performance. You can also share build scans with others, which is particularly useful if you need to ask for advice in fixing an issue with the build.
-
-[[five_things]]
-== Five things you need to know about Gradle
-
-Gradle is a flexible and powerful build tool that can easily feel intimidating when you first start. However, understanding the following core principles will make Gradle much more approachable and you will become adept with the tool before you know it.
+https://scans.gradle.com/[Build scans] provide extensive information about a build that you can use to identify issues.
+You can use build scans to identify problems with a build's performance, and even share them for debugging help.
 
-=== 1. Gradle is a general-purpose build tool
+== Concepts
```

Comment:
Should these sections be linking out to other chapters?

Reply:
I'd like to do that eventually, but right now we don't have great pages to link to about these concepts -- they're fractured over many pages. As I coalesce those fractured ideas into sections for each concept, I'll add the links here.